app/views/steps/mvp/info/edit.en.html.erb

```diff
@@ -12,8 +12,7 @@
         </h1>
 
         <p class="govuk-body">
-          Over the past few months, the User-Centred Policy Design (UCPD) team at the Ministry of Justice has been
-          looking at ways to improve the information and guidance available to young offenders.
+          Over the past few months, the User-Centred Policy Design (UCPD) team at the Ministry of Justice has been looking at ways to improve the information and guidance available to young offenders.
```

Comment:
There are many changes in this file that are not really changes just break lines.

This makes for a very difficult PR review because I have to read every single line trying to look for the change. Also mess completely the history.

Can you please only modify the line or lines that genuinely contains a real copy change?

I take you copy&paste'd whole lines, but as you can see, that produces fake changes.
We should be reading what we copy&paste not only to not change things that are not really changes, but also more importantly to not introduce typos or errors inadvertently.